src/components/profile-form/index.js

```diff
@@ -12,7 +12,8 @@ const Validators = {
   email: {regex: Regex.email, message: 'Please enter a valid email.'},
   password: {
     regex: Regex.password,
-    message: 'Please enter a password at least 8 characters long.'
+    message:
+      'Please enter a password at least 8 characters long which contains only letters (upper or lowercase) and numbers.'
```

Comment:
Lol - this was the original impetus for everything else (figuring out and documenting how to install, build, and deploy).